src/NodaTime.Demo/PeriodBuilderDemo.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2010 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NUnit.Framework;
+
+namespace NodaTime.Demo
+{
+    public class PeriodBuilderDemo
+    {
+        [Test]
+        public void ConstructionFromPeriod()
+        {
+            Period existingPeriod = Period.FromYears(5);
+            PeriodBuilder periodBuilder = Snippet.For(new PeriodBuilder(existingPeriod));
+            Period period = periodBuilder.Build();
+            Assert.AreEqual(5, period.Years);
+        }
+
+        [Test]
+        public void Build()
+        {
+            PeriodBuilder periodBuilder = new PeriodBuilder();
```

Comment:
I think it would be better to use an object initializer here:

```csharp
PeriodBuilder periodBuilder = new PeriodBuilder
{
     Years = 2,
     Months = 3,
     Days = 4
};
```